examples/data-objects/inventory-app/src/inventoryList.ts

```diff
@@ -30,7 +30,7 @@ const factory = new TypedTreeFactory({
 				quantity: 0,
 			},
 		],
-	},
+	} as any,
```

Comment:
@CraigMacomber - The cast to 'any' avoids what I assume is a bug when generating the type for FieldNode?
```
src/inventoryList.ts:22:2 - error TS2322: Type '{ parts: { name: string; quantity: number; }[]; }' is not assignable to type 'ITreeCursorSynchronous | readonly ITreeCursorSynchronous[] | { [typeNameSymbol]?: "com.contoso.app.inventory.Inventory" | undefined; parts: { ...; }; }'.
  Types of property 'parts' are incompatible.
    Property '""' is missing in type '{ name: string; quantity: number; }[]' but required in type '{ [typeNameSymbol]?: "com.contoso.app.inventory.List<Part>" | undefined; "": { [typeNameSymbol]?: "com.contoso.app.inventory.Part" | undefined; name: string | { [typeNameSymbol]?: "com.fluidframework.leaf.string" | undefined; [valueSymbol]: string; }; quantity: number | { ...; }; }[]; }'.

22  initialTree: {
    ~~~~~~~~~~~

  ../../../experimental/dds/tree2/dist/feature-libraries/schemaBuilder.d.ts:79:13
    79             [""]: NormalizeField<T, DefaultFieldKind>;
                   ~~~~
    '""' is declared here.
  ../../../experimental/dds/tree2/dist/shared-tree/schematizedTree.d.ts:54:14
    54     readonly initialTree: SchemaAware.TypedField<TRoot, SchemaAware.ApiMode.Flexible> | readonly ITreeCursorSynchronous[] | ITreeCursorSynchronous;
                    ~~~~~~~~~~~
    The expected type comes from property 'initialTree' which is declared here on type 'TypedTreeOptions<FieldSchema<Required, readonly [TreeSchema<"com.contoso.app.inventory.Inventory", { structFields: { readonly parts: FieldSchema<Required, readonly [TreeSchema<"com.contoso.app.inventory.List<Part>", { structFields: { ...; }; }>]>; }; }>]>>'


Found 1 error in src/inventoryList.ts:22
```